### ISSUE TITLE
No alternative text on any of the collection photos

### ISSUE THREAD

**Issue by devangm** [TAMULib/exhibits#109]
**URL:** https://spotlight.library.tamu.edu/spotlight/
**WCAG 2.1 AA compliant? :** No
**Violation:** 1.1.1 Non-Text Context(A)
**Issues:**  No alternative text on any of the collection photos
**Notes:** No alternative text on any of the collection photos
